app/views/patients/edit.html.erb

```diff
@@ -8,13 +8,11 @@
 </div>
 
 <div id="patient_detail" class="row">
-  <div class="col-sm-4">
-    <div id="menu" class="col-sm-9">
-      <%= render 'patients/menu', patient: @patient %>
-    </div>
+  <div class="col-sm-3">
```

Comment:
Can you do me a solid and revert these changes? We want the header that shows up in sections to line up with the LMP at intake header, and this breaks the grid. (That, and we have some javascript that requires the `menu` id; once this change goes in the tests should all pass.)